Branch-SDK/Branch-SDK/Requests/BranchShortUrlSyncRequest.m

```diff
@@ -64,6 +64,8 @@ - (BNCServerResponse *)makeRequest:(BNCServerInterface *)serverInterface key:(NS
 
 - (NSString *)processResponse:(BNCServerResponse *)response {
     if (![response.statusCode isEqualToNumber:@200]) {
+        NSLog(@"Short link creation received HTTP status code %@.", response.statusCode);
```

Comment:
I'm hoping to swap out all the `NSLog` and `logWarning` with the new `BNCLog` functions in the near future.